server/src/main/java/io/crate/execution/dml/upsert/ShardUpsertRequest.java

```diff
@@ -350,6 +363,11 @@ public Item(StreamInput in, @Nullable Streamer[] insertValueStreamers) throws IO
             if (in.readBoolean()) {
                 source = in.readBytesReference();
             }
+            if (in.getVersion().onOrAfter(Version.V_4_8_1)) {
```

Comment:
should we use `.after(Version.V_4_8_0)` instead of creating a unreleased version already?